src/responder/common/cache_req/plugins/cache_req_common.c

```diff
@@ -26,6 +26,16 @@
 #include "providers/data_provider.h"
 #include "responder/common/cache_req/cache_req_plugin.h"
 
+errno_t cache_req_idminmax_check(struct cache_req_data *data,
+	      struct sss_domain_info *domain)
+{
+   if ((domain->id_min && (data->id < domain->id_min)) ||
+       (domain->id_max && (data->id > domain->id_max))) {
+        DEBUG(SSSDBG_FUNC_DATA, "gid exceeds min/max boundaries\n");
+        return ERR_UID_OUTSIDE_RANGE;
+    }
+}
```

Comment:
This function as it is will generate a warning due to the missing return for a non-void function. IOW, you need to return EOK in case of success.